### PR TITLE
Fix #1820 Userhandler still doesn't delete some stuff

### LINKED ISSUE #1820: Userhandler still doesn't delete some stuff
Removal of the following content is missing after merging https://github.com/mybb/mybb/pull/1819:
- [x] attachments - records in `attachments` table and files
- [x] pollvotes
- [x] announcements
- [ ] ~~PMs~~ Moved to #1112
- [x] referrals (`referrer` column)

### PR DESCRIPTION
**Pull request by Sama34**
Fix #1820 Userhandler still doesn't delete some stuff

### CODE REVIEW

File: inc/datahandlers/user.php [hunk 1]
```diff
@@ -1537,6 +1544,7 @@ function delete_content($delete_uids=false)
 
 		$db->delete_query('userfields', "ufid IN({$this->delete_uids})");
 		$db->delete_query('privatemessages', "uid IN({$this->delete_uids})");
+		//$db->delete_query('privatemessages', "tuid IN({$this->delete_uids})");
```
Comment (Stefan-ST) [inline inc/datahandlers/user.php]: Why do you add a line that is commented out?

File: inc/datahandlers/user.php [hunk 2]
```diff
@@ -1458,14 +1458,22 @@ function delete_user($delete_uids, $prunecontent=0)
...
+			$db->delete_query('announcements', "uid IN({$this->delete_uids})");
 		}
 		else
 		{
 			// We're just updating the UID
+			$db->update_query('pollvotes', array('uid' => 0), "uid IN({$this->delete_uids})");
 			$db->update_query('posts', array('uid' => 0), "uid IN({$this->delete_uids})");
 			$db->update_query('threads', array('uid' => 0), "uid IN({$this->delete_uids})");
+			$db->update_query('attachments', array('uid' => 0), "uid IN({$this->delete_uids})");
+			$db->update_query('pollvotes', array('uid' => 0), "uid IN({$this->delete_uids})");
```
Comment (JN-Jones) [inline inc/datahandlers/user.php]: You added the same line twice

File: inc/datahandlers/user.php (hunk 1 above)
Comment (JN-Jones) [inline inc/datahandlers/user.php]: Should be deleted

**Comment by Sama34** [comment]
Done.

**Comment by JN-Jones** [comment]
Look fine to me. @Destroy666x @euantorano ? Anything else that we may miss here? Otherwise this can be merged.

**Pull request merged**